### ISSUE TITLE
Add accessibility option

### ISSUE THREAD

**Issue by kantord** [kiwicom/smart-faq#533]
Unfortunately, SmartFAQ doesn't meet accessibility standards yet. For example, it's very hard or impossible to use SmartFAQ without a mouse.

Given that in theory SmartFAQ should be the first point where customers enter, and direct contact with customer service should only be possible through SmartFAQ, people who are not able to use SmartFAQ due to disabilities, cannot get in contact with customer service and cannot use SmartFAQ or can only use it with difficulty.

*Everyone*, regardless of their ability *has to* be able to get help from Kiwi.com. Therefore, until the accessibility of SmartFAQ is improved to meet stricter standards, we *need* to provide an accessibility option that let's people with difficulties in using SmartFAQ get in direct contact with customer service. Because these customers cannot currently use SmartFAQ, they have to be prioritized over customers who can. So the accessibility option has to be designed in a way that only people with difficulties will use it.
